provision/acc_provision/acc_provision.py

```diff
@@ -37,15 +38,22 @@
 
 
 FLAVORS_PATH = os.path.dirname(os.path.realpath(__file__)) + "/flavors.yaml"
+VERSIONS_PATH = os.path.dirname(os.path.realpath(__file__)) + "/versions.yaml"
+
 
+with open(VERSIONS_PATH, 'r') as stream:
+    try:
+        doc = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
```

Comment:
standard logger func for error is "err". We should use "err" instead of print.
suggesting the same for line 55

Reply:
Good catch.